ffi/examples/read-table/schema.h

```diff
@@ -2,6 +2,7 @@
 #include "read_table.h"
 #include "kernel_utils.h"
 #include <stdint.h>
+#include <time.h>
```

Comment:
do we need the time header?

Reply:
hmm I got this error otherwise
```
/home/kssenii/workspace/ClickHouse2/contrib/delta-kernel-rs/ffi/examples/read-table/read_table.c:168:24: error: implicit declaration of function ‘gmtime’ [-Wimplicit-function-declaration]
  168 |   struct tm *tm_info = gmtime(&tv.tv_sec);
      |                        ^~~~~~
/home/kssenii/workspace/ClickHouse2/contrib/delta-kernel-rs/ffi/examples/read-table/read_table.c:9:1: note: ‘gmtime’ is defined in header ‘<time.h>’; this is probably fixable by adding ‘#include <time.h>’
    8 | #include "schema.h"
  +++ |+#include <time.h>
    9 | #include "kernel_utils.h"
```
(just noticed I should have added it in read_table.c instead of schema.h, but anyway that fixed the problem).
I will remove this header, would leave it for myself locally only.